spec/models/user_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+describe User, type: :model do
+  let(:user) { FactoryBot.build(:random_user) }
```

Comment:
I usually use subject here like this:
```
subject { create(:connection) }
```
Then, in the tests you can do:
```
it { is_expected.to validate_presence_of(:name) }
```
Also, it is not necessary to call `FactoryBot` before `create` and `build` methods because it should be already loaded for the test files I believe.